sdk/cognitivelanguage/azure-ai-language-questionanswering/README.md

```diff
@@ -0,0 +1,261 @@
+[![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/azure-sdk-for-python.client?branchName=main)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=46?branchName=main)
+
+# Azure Cognitive Language Services Question Answering client library for Python
+
+Question Answering is a cloud-based API service that lets you create a conversational question-and-answer layer over your existing data. Use it to build a knowledge base by extracting questions and answers from your semi-structured content, including FAQ, manuals, and documents. Answer users’ questions with the best answers from the QnAs in your knowledge base—automatically. Your knowledge base gets smarter, too, as it continually learns from user behavior.
+
+[Source code][questionanswering_client_src] | [Package (PyPI)][questionanswering_pypi_package] | [API reference documentation][questionanswering_refdocs] | [Product documentation][questionanswering_docs] | [Samples][questionanswering_samples]
+
+## Getting started
+
+### Prerequisites
+
+* Python 2.7, or 3.6 or later is required to use this package.
+* An [Azure subscription][azure_subscription]
+* An existing Question Answering resource
+
+> Note: the new unified Cognitive Language Services are not currently available for deployment.
+
+### Install the package
+
+Install the Azure QuestionAnswering client library for Python with [pip][pip_link]:
+
+```bash
+pip install azure-ai-language-questionanswering --pre
+```
+
+### Authenticate the client
+
+In order to interact with the Question Answering service, you'll need to create an instance of the [`QuestionAnsweringClient`][questionanswering_client_class] class. You will need an **endpoint**, and an **API key** instantiate a client object. For more information regarding authenticating with Cognitive Services, see [Authenticate requests to Azure Cognitive Services][cognitive_auth].
+
+#### Get an API key
+
+You can get the **endpoint** and an **API key** from the Cognitive Services resource or Question Answering resource in the [Azure Portal][azure_portal].
```

Comment:
Does QNA support multi-service / Cognitive Services resource? Wondering about it since above it says we need an existing Questioning Answering resource

Reply:
hmm I'm not sure - I will look into this!